hlxsites/merative2
# chore(logo): add update curam logo

## Issue

Fixes N/A

## Description

**New**

- This PR adds the SVG of the new Curam logo
- This is in preparation for the Curam branding launch that we need to implement within Franklin for the **6/20** rollout.

## Design Specs

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=496%3A6550&t=8L51Ja1X4Y3QXsj2-1

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/drafts/putra/curam
- After (Changes from this PR): https://chore-curam-logo-update--merative2--proeung.hlx.page/drafts/putra/curam
  
## Testing Instruction

- Visit this draft/copy of the `Curam` tab of the masthead/nav - https://chore-curam-logo-update--merative2--proeung.hlx.page/drafts/putra/curam
- Add `:curam:` shortcode to the doc and ensure that the new Curam logo is displaying correctly.
